src/select-travel-token-screen/SelectTravelTokenScreenComponent.tsx

```diff
@@ -272,7 +272,7 @@ export const SelectTravelTokenScreenComponent = ({onAfterSave}: Props) => {
               onPress={onSave}
               text={t(TravelTokenTexts.toggleToken.saveButton)}
               interactiveColor="interactive_0"
-              disabled={!!selectedToken}
+              disabled={!selectedToken}
```

Comment:
It was with one `!` from before, but during my fixes and tests I added a `!!` double, so basically I am reverting it.